jax/numpy/lax_numpy.py

```diff
@@ -2617,6 +2617,33 @@ def polyadd(a, b):
 
   return lax.add(a,b)
 
+def _trim_zeros(a):
```

Comment:
Slighly cleaner I think:
```python
def _trim_zeros(a):
  for i, v in enumerate(a):
    if v != 0:
      return a[i:]
  return a[:0]
```

Reply:
yes this looks much better than what I had.

Thanks